<?xml version="1.0" ?>
 
<robot xmlns:xacro="http://wiki.ros.org/xacro">

    <ros2_control name="RobotSystem" type="system">
        <hardware>
            <xacro:if value="${use_gazebo}">
                <plugin>gazebo_ros2_control/GazeboSystem</plugin>
            </xacro:if>
        </hardware>
 
        <joint name="joint2">
            <command_interface name="position">
                <param name="min">-2.879793</param>
                <param name="max">2.879793</param>
            </command_interface>
            <state_interface name="position"/>
        </joint>
        
 
        <joint name="joint3">
            <command_interface name="position">
                <param name="min">-2.879793</param>
                <param name="max">2.879793</param>
            </command_interface>
            <state_interface name="position"/>
        </joint>
 
        <joint name="joint4">
            <command_interface name="position">
                <param name="min">-2.879793</param>
                <param name="max">2.879793</param>
            </command_interface>
            <state_interface name="position"/>
        </joint>
 
        <joint name="joint5">
            <command_interface name="position">
                <param name="min">-2.879793</param>
                <param name="max">2.879793</param>
            </command_interface>
            <state_interface name="position"/>
        </joint>


    </ros2_control>    
</robot>